src/frontend/src/lib/types/transaction.ts

```diff
@@ -28,3 +32,7 @@ export type TransactionUiCommon = Pick<Transaction, 'blockNumber' | 'from' | 'to
 	toExplorerUrl?: string;
 	fromExplorerUrl?: string;
 };
+
+export type UnifiedTransactionUi = (BtcTransactionUi | EthTransactionUi | IcTransactionUi) & {
```

Comment:
We have "Activity", "AllTransactions" and "UnifiedTransactions". For maintenance reasons it would be maybe interesting to find a common denominator?

Reply:
I will call as following:

`Activity` --> `AllTransactions`
`AllTransactions` --> `AllTransactionsList`
type `UnifiedTransactionUi` --> `AllTransactionsUi`

Reply:
That said, sounds good. It's easy to refactor a name anyway, thanks!